### PR TITLE
Added README

### PR DESCRIPTION
**Pull request by frankk74**
Hopefully this pull request is okay. All I did was create a readme file based on what was on the BGG forum.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+  --clean_xml           Clear out local xml cache. (default=Off)
+```
+
+## Authors
+
+Contributors names and contact info
+
+* Daniel Shourd
+** [on BGG](https://boardgamegeek.com/user/RKDN)
+** [her on github](https://github.com/RKDN)
```
Comment (kberg) [inline README.md:55]: I wouldn't know better but it's not clear Daniel uses 'her' as a pronoun.

File: README.md [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+## Version History
+
+* 0.2
+    * Various bug fixes and optimizations
+* 0.1
+    * Initial Release
+
+## License
+
+This project is licensed under the [NAME HERE] License - see the LICENSE.md file for details
```
Comment (kberg) [inline README.md:66]: Since there is no license, /shrug?

**Comment by RKDN** [comment]
I'm going to reject this for now because I'd rather not have my BGG linked to my github page, some of this information is a bit out of date already, and I'm not sure of a license yet.

**Review by RKDN — CHANGES_REQUESTED** [review]
Update this with the new instructions from the BGG post and put the MIT license where [NAME HERE] is and I will approve.

**Pull request merged**